TradeItIosTicketSDK2/TradeItYahooLinkBrokerUIFlow.swift

```diff
@@ -26,20 +28,37 @@ import SafariServices
     ) {
         self.oAuthCallbackUrl = oAuthCallbackUrl
 
-        guard let selectBrokerViewController = self.viewControllerProvider.provideViewController(forStoryboardId: .yahooBrokerSelectionView) as? TradeItYahooBrokerSelectionViewController else {
+        guard let brokerSelectionViewController = self.viewControllerProvider.provideViewController(forStoryboardId: .yahooBrokerSelectionView) as? TradeItYahooBrokerSelectionViewController else {
             print("TradeItSDK ERROR: Could not instantiate TradeItYahooBrokerSelectionViewController from storyboard!")
             return
         }
 
-        selectBrokerViewController.oAuthCallbackUrl = oAuthCallbackUrl
+        brokerSelectionViewController.oAuthCallbackUrl = oAuthCallbackUrl
+        brokerSelectionViewController.delegate = self.delegate
 
         if (asRootViewController) {
-            navController.setViewControllers([selectBrokerViewController], animated: true)
+            navController.setViewControllers([brokerSelectionViewController], animated: true)
         } else {
-            navController.pushViewController(selectBrokerViewController, animated: true)
+            navController.pushViewController(brokerSelectionViewController, animated: true)
         }
     }
 
+    func presentLinkBrokerFlow(
+        fromViewController viewController: UIViewController,
+        showWelcomeScreen: Bool = true,
+        showOpenAccountButton: Bool = true,
+        oAuthCallbackUrl: URL,
+        delegate: YahooLauncherDelegate?
+        ) {
```

Comment:
Is it possible to alter the existing function signature to include `delegate` instead of creating this new function?

Reply:
I was trying to avoid changing the protocol.